#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: eOVPN\n"
"POT-Creation-Date: 2021-02-17 16:10+0530\n"
"PO-Revision-Date: 2021-02-17 16:04+0530\n"
"Last-Translator: Jagadeesh Kotra <jagadeesh.01101011@gmail.com>\n"
"Language-Team: \n"
"Language: en\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Generator: Poedit 2.4.2\n"
"X-Poedit-Basepath: ..\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Poedit-KeywordsList: __push_to_statusbar\n"
"X-Poedit-SourceCharset: UTF-8\n"
"X-Poedit-SearchPath-0: eovpn\n"
"X-Poedit-SearchPath-1: data\n"
"X-Poedit-SearchPath-2: data/ui\n"
"X-Poedit-SearchPathExcluded-0: __pycache__/*\n"

#: data/com.github.jkotra.eovpn.desktop:3
#: data/com.github.jkotra.eovpn.desktop:4
msgid "eOVPN"
msgstr ""

#: data/com.github.jkotra.eovpn.desktop:5
msgid "OpenVPN Configuration Manager"
msgstr ""

#: data/ui/about.glade:12
msgid "OpenVPN Configuration(s) Manager"
msgstr ""

#: data/ui/log.glade:7 data/ui/main.glade:83
msgid "Log"
msgstr ""

#: data/ui/log.glade:50
msgid "Close"
msgstr ""

#: data/ui/main.glade:19
msgid "OpenVPN is running in background!"
msgstr ""

#: data/ui/main.glade:20
msgid ""
"OpenVPN is still running in background. It must be killed before attempting "
"to connect again. Kill it?"
msgstr ""

#: data/ui/main.glade:32 data/ui/settings.glade:561
msgid "Cancel"
msgstr ""

#: data/ui/main.glade:46
msgid "Kill"
msgstr ""

#: data/ui/main.glade:99
msgid "Update"
msgstr ""

#: data/ui/main.glade:117 data/ui/settings.glade:503
msgid "Settings"
msgstr ""

#: data/ui/main.glade:135
msgid "About"
msgstr ""

#: data/ui/main.glade:153
msgid "Exit"
msgstr ""

#: data/ui/main.glade:190 eovpn/main_window.py:211
msgid "Disconnected"
msgstr ""

#: data/ui/main.glade:236
msgid "IP: 127.0.0.1"
msgstr ""

#: data/ui/main.glade:286
msgid "Location: Earth"
msgstr ""

#: data/ui/main.glade:374
msgid "TCP/UDP"
msgstr ""

#: data/ui/main.glade:448
msgid "OpenVPN configuration"
msgstr ""

#: data/ui/main.glade:474 eovpn/main_window.py:218
msgid "Connect!"
msgstr ""

#: data/ui/settings.glade:58
msgid "Settings Saved"
msgstr ""

#: data/ui/settings.glade:115
msgid "General"
msgstr ""

#: data/ui/settings.glade:148
msgid "Configuration Source: "
msgstr ""

#: data/ui/settings.glade:162
msgid "https://example.com/configs.zip"
msgstr ""

#: data/ui/settings.glade:173
msgid "Validate"
msgstr ""

#: data/ui/settings.glade:202
msgid "Update configuration(s) on start"
msgstr ""

#: data/ui/settings.glade:243
msgid "Requires Authorization"
msgstr ""

#: data/ui/settings.glade:271
msgid "Username:"
msgstr ""

#: data/ui/settings.glade:309
msgid "Password:"
msgstr ""

#: data/ui/settings.glade:357
msgid "Authorization"
msgstr ""

#: data/ui/settings.glade:379
msgid "OpenVPN CRT:"
msgstr ""

#: data/ui/settings.glade:476
msgid "Connect on Launch"
msgstr ""

#: data/ui/settings.glade:507
msgid "Save"
msgstr ""

#: data/ui/settings.glade:519
msgid "Reset"
msgstr ""

#: data/ui/settings.glade:547
msgid "Are you sure?"
msgstr ""

#: data/ui/settings.glade:548
msgid "All settings will be permanently deleted."
msgstr ""

#: data/ui/settings.glade:576
msgid "Yes"
msgstr ""

#: eovpn/main_window.py:112
msgid "Failed to connect!"
msgstr ""

#: eovpn/main_window.py:177
msgid "No Network"
msgstr ""

#: eovpn/main_window.py:193
msgid "Connected"
msgstr ""

#: eovpn/main_window.py:206
msgid "Disconnect!"
msgstr ""

#: eovpn/main_window.py:284
msgid "No config selected."
msgstr ""

#: eovpn/openvpn.py:230
msgid "Connecting.."
msgstr ""

#: eovpn/openvpn.py:236
msgid "Config requires authorization (auth-user-pass)"
msgstr ""

#: eovpn/openvpn.py:252
msgid "Connected to {}."
msgstr ""

#: eovpn/openvpn.py:271
msgid "Disconnected."
msgstr ""

#: eovpn/openvpn.py:287
msgid "OpenVPN not found."
msgstr ""

#: eovpn/openvpn.py:360
msgid "Invalid Remote"
msgstr ""

#: eovpn/openvpn.py:365
msgid "Updating..."
msgstr ""

#: eovpn/openvpn.py:372 eovpn/openvpn.py:398
msgid "Config(s) updated!"
msgstr ""

#: eovpn/openvpn.py:380
msgid "No config(s) found!"
msgstr ""

#: eovpn/openvpn.py:416
msgid "Success"
msgstr ""

#: eovpn/openvpn.py:416
msgid "Valid Remote"
msgstr ""

#: eovpn/openvpn.py:417
msgid "{} OpenVPN configuration's found."
msgstr ""

#: eovpn/openvpn.py:421
msgid "Validate Error"
msgstr ""

#: eovpn/openvpn.py:421
msgid "Error"
msgstr ""
